gym_solo/envs/solo8v2vanilla.py

```diff
@@ -39,37 +39,33 @@ class Solo8VanillaConfig(Solo8BaseConfig):
   }
 
 
-class Solo8VanillaEnv(gym.Env):
+class Solo8VanillaEnv(Solo8BaseEnv):
   """An unmodified solo8 gym environment.
   
   Note that the model corresponds to the solo8v2.
   """
   def __init__(self, use_gui: bool = False, realtime: bool = False, 
-               config=None, **kwargs) -> None:
```

Comment:
Why remove the None?

Reply:
lol technically returning None is implied so it doesn't really matter... vscode puts it in but I don't when I code--do you have a preference for if we need to keep it or not?